packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -889,7 +889,7 @@ const ReactDataGrid = React.createClass({
       onRowHover: this.onRowHover,
       getDataGridDOMNode: this.getDataGridDOMNode,
       isScrollingVerticallyWithKeyboard: this.isKeyDown(40) || this.isKeyDown(38), // up or down
-      isScrollingHorizontallyWithKeyboard: this.isKeyDown(37) || this.isKeyDown(39) // left or right
+      isScrollingHorizontallyWithKeyboard: this.isKeyDown(37) || this.isKeyDown(39) || this.isKeyDown(9) // left or right or tab
```

Comment:
use https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid/src/KeyCodes.js instead of the crazy numbers and it's good to go